src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -346,99 +341,141 @@ public void fillUpExistingHoneyFrameFromHive(Date currentDate) {
         Random random = new Random();
         System.out.println();
         List<HoneyFrame> honeyFrames = this.getHoneyFrames();
```

Comment:
Remove `List<HoneyFrame> honeyFrames = this.getHoneyFrames();` and use `honeyFrames` everywere instead of `this.honeyFrames`.

Reply:
done